presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/LambdaCaptureDesugaringRewriter.java

```diff
@@ -12,13 +12,16 @@
  * limitations under the License.
  */
 
-package com.facebook.presto.sql.planner;
+package com.facebook.presto.sql.planner.iterative.rule;
```

Comment:
keep such changes as separate commits. Please do not mix logical changes with refactoring changes in the same commit.

Reply:
Sorry for not asking before, but can you please tell me why this package is a better place for this class?

Reply:
This rewriter is used by deprecated `DesugarignOptimizer` in `planner.optimizers` and `DesugarLambdaExpression` in `planner.iterative.rule`. After completely moving to the iterative optimizer, I don't think it's gonna be used outside of rules, so I moved it there.